### PR TITLE
Fixes for Yappi and Django >= 1.5

### LINKED ISSUE #1: Recent changes in yappi do break this app
The recent backward incompatible changes in yappi (including constant changes) do break this application...
See: https://bitbucket.org/sumerc/yappi/changesets

I got it running using this version (tag 0.62 on bitbucket):
https://bitbucket.org/sumerc/yappi/src/e58f9ea157ef
if you use pip, try:
```pip install hg+ssh://hg@bitbucket.org/sumerc/yappi@0.62#egg=yappi```

However not everything works... due to some other changes sometimes the following Exceptions is being catched:

  * YStats instance has no attribute 'index'

**BEWARE OF THE VERSION 0.62 on PyPI and googlecode**,
since they are not the same as the 0.62-tagged version on bitbucket and already include the contant changes.

Yappi seems to be under heavy development right now, so maybe other breaking changes will follow.
It would be great to have a requiremets.txt so the dependencies could be installed via pip.

For my runing version i used:
```
pip install pympler psutil
pip install git+git://github.com/daevaorn/django-query-exchange.git@8c34058401b2f44eabb195ada4c744001f686047#egg=django-query-exchange
pip install hg+ssh://hg@bitbucket.org/sumerc/yappi@0.62#egg=yappi
```

So an example requirements.txt file would be:
----
```
Pympler==0.2.1
psutil==0.4.1
-e git+git://github.com/daevaorn/django-query-exchange.git@8c34058401b2f44eabb195ada4c744001f686047#egg=django-query-exchange
-e hg+ssh://hg@bitbucket.org/sumerc/yappi@0.62#egg=yappi
```

### PR DESCRIPTION
**Pull request by aluminiumgeek**
Fixes #1, #3

### CODE REVIEW

File: profiling_dashboard/templates/profiling_dashboard/base.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" href="http://twitter.github.com/bootstrap/1.4.0/bootstrap.min.css">
+    <link rel="stylesheet" href="/static/bootstrap.min.css">
```
Comment (kmike) [inline profiling_dashboard/templates/profiling_dashboard/base.html]: Shouldn't it use `{% static %}` template tag?

File: setup.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
     if cmd in sys.argv:
         from setuptools import setup
 
-version='0.1'
+version='0.2'
```
Comment (kmike) [inline setup.py]: Could you please remove the version number update from the PR?

**Comment by kmike** [comment]
Thanks! I haven't actually tried it, but the PR looks good.

Could you please also add yourself to https://github.com/kmike/django-profiling-dashboard/blob/master/AUTHORS.rst?

FYI: I have no plans to continue django-profiling-dashboard development myself, but I'm fine with making a release with your changes if you think the app is useful.

File: setup.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
     description = """ Django profiling dashboard for debugging CPU, memory and other resources usage in live servers """,
 
     long_description = open('README.rst').read(),
-    requires = ['django (>= 1.3)', 'yappi (>= 0.54)', 'psutil (>= 0.4.1)', 'pympler (>= 0.2.1)', 'query_exchange'],
+    requires = ['django (>= 1.5)', 'yappi (>= 0.54)', 'psutil (>= 0.4.1)', 'pympler (>= 0.2.1)', 'query_exchange'],
```
Comment (kmike) [inline setup.py]: README should be also updated - it says 'django >= 1.3' is required.

**Comment by aluminiumgeek** [comment]
Sorry for delay :)
I've updated the code.

File: profiling_dashboard/urls.py [hunk 4]
```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import
-from django.conf.urls.defaults import *
+#from __future__ import absolute_import
```
Comment (kmike) [inline profiling_dashboard/urls.py]: could you please keep this import?

File: profiling_dashboard/templates/profiling_dashboard/base.html [hunk 5]
```diff
@@ -1,7 +1,9 @@
+{% load staticfiles %}
+
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" href="http://twitter.github.com/bootstrap/1.4.0/bootstrap.min.css">
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
```
Comment (kmike) [inline profiling_dashboard/templates/profiling_dashboard/base.html]: I think the css file should go in `/profiling_dashboard/css/bootstrap.min.css` - this way it won't conflict with user static files.

**Comment by kmike** [comment]
Thanks! I have a few more comments :)

File: profiling_dashboard/templates/profiling_dashboard/base.html (hunk 5 above)
Comment (aluminiumgeek) [inline profiling_dashboard/templates/profiling_dashboard/base.html]: You're right

**Comment by aluminiumgeek** [comment]
Fixed.

**Comment by kmike** [comment]
Thanks @aluminiumgeek!

**Comment by kmike** [comment]
0.2 is on pypi

**Pull request merged**